.hadolint.yaml

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
Ok. I believe that should be possible. I'd assume we are using the binary and we do not modify sources. It's no different than when we use linux kernel for example. Is there any paper-work needed to ask for exemption? I will gladly follow that as hadolint looks rather nice.

Reply:
https://issues.apache.org/jira/browse/LEGAL-447 was where I asked about an LGPL lib in a similar case. I don't know (haven't checked) if GPL is any different in this case.

I think my thinking last night was "GPL" is a bit more wide-reaching than LGPL so we might have to double check.

From   http://www.apache.org/legal/resolved.html#build-tools

> The most important criteria to be included in this category is that such tools are not customarily part of distributions of running code, unless such deliverables include source anyway. Other criteria for inclusion include: having been around for years, being defacto standards in their respective communities, being made available under are under "library" or "lesser" licenses or otherwise containing an exception which ensures that usage of this tool does not affect the license of the code against which it is run.

Reply:
Cool. I opened https://issues.apache.org/jira/browse/LEGAL-450 . If I don't hear from them soon, I might extract it out to a separate PR to not be blocked on it.

Reply:
That one was super quick: 

> As is pointed out in our FAQ: https://apache.org/legal/resolved.html#prohibited it is OK to use GPL build tools provided they leave no code traces (licensed under a cat X license) in your final release artifacts.

So I think we are safe :)

https://issues.apache.org/jira/browse/LEGAL-450?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=16796240#comment-16796240